components/headers/MainHeader.style.ts

```diff
@@ -0,0 +1,40 @@
+import device from '@utils/breakpointsDevice';
+import styled from 'styled-components';
+
+export const LogoContainer = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  cursor: pointer;
+`;
+
+export const LogoImg = styled.img`
+  width: 28.797px;
+  height: 33.069px;
+`;
+
+export const LogoText = styled.img`
+  width: 80px;
+  height: 22px;
+
+  @media ${device.mobile} {
+    display: none;
+  }
+`;
+
+export const ButtonContainer = styled.div`
+  display: flex;
+  gap: 36px;
+
+  @media ${device.mobile} {
+    gap: 20px;
+  }
+`;
+
+export const Button = styled.button`
+  color: var(--black-black_333236, #333236);
```

Comment:
```suggestion
  color: var(--black200);
```
`globals.css` 의 색상 팔레트 참고 부탁드립니다